tutorials/xr/xr_full_screen_effects.rst

```diff
@@ -0,0 +1,81 @@
+.. _doc_xr_full_screen_effects:
+
+XR full screen effects
+======================
+
+When adding custom full screen effects to your XR application, one valid approach is
+applying a full screen quad to the XR camera and applying effects to that quad's shader.
+
+.. image:: img/xr_full_screen_effects_starting_quad.webp
+
+Here is a simple vignette shader that you might apply to the quad, darkening the edges
+of a user's screen:
```

Comment:
If we're not linking to https://docs.godotengine.org/en/latest/tutorials/shaders/advanced_postprocessing.html, then I think we need to include some more information than this, explaining:

1. The `MeshInstance3D` node they need to add, the size of the quad mesh, and to put it as a child of the camera. Some of this is shown in the images, but I think we should have it in words too.
2. That we're making it full screen by setting the `POSITION` in the shader. (This ends up being implied in the "Applying the projection matrix" section, but I feel like it'd be best to state it up front)